src/Sarif.Driver/Sdk/SdkResources.resx

```diff
@@ -203,4 +203,11 @@
   <data name="ERR0997_NoValidAnalysisTargets" xml:space="preserve">
     <value>No valid analysis targets were specified.</value>
   </data>
+  <data name="ERR0997_ExceptionAccessingFile" xml:space="preserve">
+    <value>An exception was raised accessing a file specified on the command-line ('{0}'). Exception information:
```

Comment:
Also, I don't think you need to both quote and parenthesize the file name. Maybe:

```
An error occurred while attempting to access a file specified on the command line: '{0}'
Exception information:
{1}
```